addons/core/translations/en-us.yaml

```diff
@@ -154,7 +154,7 @@ worker-filter-generator:
     helper: Format a worker tag
   name:
     label: Name
-    helper: Format a worker name
+    helper: Format a worker name using regex
```

Comment:
Isn't regex only valid if a user chooses the `matches` keyword though? Or am I mistaken?

Reply:
yeah i see ur point, the wording could imply that regex can be used with all operators.

Reply:
Yup adding the wording change in the dropdown works.
If the user chooses `matches (regex)` then it should still be `"/name" matches ""`
Also side-note: `includes` should actually be `contains`